playwright/all_tests/assignment/assignmentFilters.spec.ts

```diff
@@ -38,11 +38,11 @@ test.describe('Assignment filter tests', () => {
     await page.getByTestId('contentType-multi-select-ready-button').click()
 
     await page.getByTestId('aihe').click()
-    // vaikuttaminen
-    await page.getByTestId('aihe-option-007').click()
-    // ympärisöt
-    void page.getByTestId('aihe-option-008').click()
-    await checkResponseAfterFiltering(page, 'SUKO')
+    // Pohjoismaat
+    void page.getByTestId('aihe-option-013').click()
+
+    expect(await checkResponseAfterFiltering(page, 'SUKO')).toHaveLength(1)
+
```

Comment:
Playwrightin omien dokkareiden mukaan nää clickin jälkeen waittaamiset kannattais tehdä niin päin, että ensin tehdään wait-promise, sit klikataan awaitin kanssa, ja sit awaitataan se wait-promise. Itsestäkin tuntuis varmemmalta noin päin, ku muuten se klikkaus saattaa periaatteessa ehtiä maaliin ennen kuin waitti on viritetty. Tyyliin:

```
const checkResponseAfterFilteringPromise = checkResponseAfterFiltering(page, 'SUKO')
await page.getByTestId('aihe-option-013').click()
expect(await checkResponseAfterFilteringPromise).toHaveLength(1)
```

Tää vanha tyyli tuntuu kyl toimivan, ni ei varmaan kauhee kiire muuttaa tota joka paikkaan, mut ehkä vois uudet tehdä tolleen dokumentaation mukaisesti.

Reply:
Length-checkiä parempi ois katsoa, että sivulla näkyy odotetun nimiset tehtävät (ja vain odotetun nimiset tehtävät eikä muita)